pkg/graveler/committed/apply.go

```diff
@@ -26,99 +26,129 @@ func incrementDiffSummary(d *graveler.DiffSummary, typ graveler.DiffType) {
 
 // ReferenceType represents the type of the reference
 
-// applyFromSource applies all changes from source to writer.
-func applyFromSource(ctx context.Context, logger logging.Logger, writer MetaRangeWriter, source Iterator) error {
+// applyFromIter applies all changes from Iterator to writer.
+// returns the number of writes
```

Comment:
```suggestion
// applyFromIter applies all changes from iter to writer and returns the number of writes
```

(not sure what "number of writes" means)